source/versioned-api/tests/crud-api-version-1-strict.json

```diff
@@ -154,6 +154,11 @@
                 "$limit": 1
               }
             ]
+          },
```

Comment:
The name of this test is still "aggregate on database appends declared API version" which is inaccurate as it's also checking that the correct error is generated when an unsupported stage is used in a `db.aggregate()`. I used the same name as before because "distinct appends declared API version" does the same thing. Let me know if I should change the name of both tests.

Reply:
I think the name is still okay.